climada/entity/exposures/gpw_import.py

```diff
@@ -1,8 +1,22 @@
-# -*- coding: utf-8 -*-

```

Comment:
I removed this line from all files. This is a relict from Python 2 and not used in Python 3.